includes/db.inc

```diff
@@ -35,9 +35,14 @@ function islandora_usage_stats_pid_id($pid) {
  */
 function islandora_usage_stats_pid_to_db($pid) {
```

Comment:
@bryjbrown Did you happen to miss the bit after the code snippet, describing the change below ("where we have an object to pass")? (could change the parameter name we use as well, if desired).

As I'm imagining it, it would be _in addition_ to throwing exceptions... It just eliminates (or tries to) the possibility of throwing an exception for an object we have loaded elsewhere in memory.